integration/templates/ingress_controller_basic_values.go

```diff
@@ -10,14 +10,14 @@ controller:
   k8sAppLabel: nginx-ingress-controller
   metricsPort: 10254
 
-  replicas: 1
+  replicas: 3
 
   configmap:
     name: ingress-nginx
 
   image:
     repository: giantswarm/nginx-ingress-controller
-    tag: 0.12.0
+    tag: 0.23.0
```

Comment:
I think this is wrong. We are overriding the settings but we should just use the defaults.

Having to bump the tag here too is error prone. Can you try it without this?

Reply:
I tried it, and it results an error like below, 
```
unknown flag: --enable-dynamic-certificates```
So we should keep it as it 0.23.0

Reply:
And this part is almost everything in this PR 😆